test/LostConnectionTest.js

```diff
@@ -47,7 +47,7 @@ describe('Lost connection', function () {
     });
 
     it('M2 starts, M1 goes down, client sets M2 as owner', function (done) {
-        this.timeout(12000);
+        this.timeout(32000);
```

Comment:
why dont we have a global timeout ? Like 1 minute maybe and use it everywhere? These timeouts are hard to justify in my opinion. They seem random values.

Most of the time if a test fails in x seconds it will also fail in 2x seconds too.

Reply:
At first I was doing this because default mocha timeout was too short for only a handful of tests we had and majority of tests passing fine with the default timeout. Also, this way we would now if a supposedly short term test took a lot of time (e.g. simple map.put taking a few seconds). However, I agree specifying timeout for individual tests is causing more pain than it solves. So I guess we can put a global large timeout. Can you create an issue for that?

Reply:
I created the issue #452.